target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -299,6 +299,7 @@ tplink,wbs510-v2)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "green:link3" "wlan0" "60" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "green:link4" "wlan0" "80" "100"
 	;;
+meraki,mr12|\
```

Comment:
You should move the whole block to match alphabetic sorting.

Reply:
Done.